packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/DbSyncNetworkInfoProvider.ts

```diff
@@ -9,25 +9,19 @@ import {
   SupplySummary,
   createSlotEpochCalc
 } from '@cardano-sdk/core';
-import { DbSyncProvider, DbSyncProviderDependencies } from '../../util/DbSyncProvider';
-import { Disposer, EpochMonitor } from '../../util/polling/types';
-import { GenesisData } from './types';
-import { InMemoryCache, UNLIMITED_CACHE_TTL } from '../../InMemoryCache';
+import { DbSyncProvider, DbSyncProviderDependencies, Disposer, EpochMonitor } from '../../util';
+import { GenesisData, InMemoryCache, UNLIMITED_CACHE_TTL } from '../..';
 import { Logger } from 'ts-log';
 import { NetworkInfoBuilder } from './NetworkInfoBuilder';
 import { RunnableModule } from '@cardano-sdk/util';
-import { loadGenesisData, toGenesisParams, toLedgerTip, toProtocolParams, toSupply } from './mappers';
+import { toGenesisParams, toLedgerTip, toProtocolParams, toSupply } from './mappers';
 import memoize from 'lodash/memoize';
 
 /**
  * Properties that are need to create DbSyncNetworkInfoProvider
  */
-export interface NetworkInfoProviderProps {
-  /**
-   * Cardano node config path string
-   */
-  cardanoNodeConfigPath: string;
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface NetworkInfoProviderProps {}
```

Comment:
Maybe just delete it (and the 1st constructor argument)?

Reply:
Yes, I thought the same.
The reason why I wrote this is to let the provider to respect the same signature of all our providers:
`ProviderName(configuration, dependencies)`

Reply:
0173f5190 should solve this